src/MultiRegion/multi_region_field.jl

```diff
@@ -70,6 +70,8 @@ Base.size(f::GriddedMultiRegionField) = size(getregion(f.grid, 1))
 Reconstruct a global field from `mrf::MultiRegionField` on the `CPU`.
 """
 function reconstruct_global_field(mrf::MultiRegionField)
+
+    # TODO: Is this correct? Shall we reconstruct a global field on the architecture of the grid?
```

Comment:
`arch` should be an argument to `reconstruct_global_field` if we need to do this on CPU for some reason

Reply:
agreed